sys-kernel/coreos-kernel/files/amd64_defconfig-4.0

```diff
@@ -1965,6 +1974,7 @@ CONFIG_MOUSE_PS2=m
 # CONFIG_MOUSE_PS2_ELANTECH is not set
 # CONFIG_MOUSE_PS2_SENTELIC is not set
 # CONFIG_MOUSE_PS2_TOUCHKIT is not set
+CONFIG_MOUSE_PS2_FOCALTECH=y
```

Comment:
Is this pulled in automatically or is it optional? We don't have any other special mice